gulp/tasks/s3-paths.js

```diff
@@ -16,11 +16,10 @@ gulp.task('s3Paths', function () {
 
         gulp.src(filename)
 
-            .pipe(replace(/[ ](href|src)="(?!http)/g, ' $1="' + baseUrl + '/'))
-            .pipe(replace(/(url\(")/g, ' $1' + baseUrl))
-
             // Base tag, needed for ui-router
             .pipe(replace('<base href="/">', '<base href="' + baseUrl + '">'))
+            .pipe(replace(/[ ](href|src)="(?!http)/g, ' $1="' + baseUrl))
```

Comment:
A small note here: the intent of this line is to change all relative links to absolute links, e.g. `<img src="images/art.jpg" />` to `<img src="https://anno.tate.org.uk/images/art.jpg"` and, with this PR update, it only really works if the href/src _doesn't_ start with a slash.

I want this in writing because it took me a while to understand that "negative lookahead" Regex code and I'll be darned if I forget it.

Reply:
I still don't completely understand this regex.